GOVERNANCE.md

```diff
@@ -17,10 +17,12 @@ __TLDR; Project decisions are made by a consensus of the projects steering commi
 
 The FOSSBilling community is made up of individuals, companies, or groups who use and/or contribute to the FOSSBilling software.
 
-Anybody is free to use the core FOSSBilling software within the terms of the [license](https://github.com/FOSSBilling/FOSSBilling/blob/main/LICENSE). There are not and never will be any fees for usage, and no special requirements.
+Anybody is free to use the core FOSSBilling software within the terms of the [license](https://github.com/FOSSBilling/FOSSBilling/blob/main/LICENSE). There are not and never will be any fees for usage, and no special requirements. 
 
 We really appreciate user contributions but if you just want to download, install, and go, then you have the complete freedom to do that.
 
+Third-party extensions to FOSSBilling are also welcomed, it would be appreciated if these were discussed and informed by the community, but this is not mandatory. In these cases please do be aware of the restrictions attached to usage of the [FOSSBilling name and branding](https://github.com/FOSSBilling/branding#readme). 
```

Comment:
And this part also just feels a bit strange to me, it's worded a bit confusingly and I'm not entirely sure what purpose it serves

Reply:
Reworded - the purpose was essentially drawing attention to branding restrictions for third-party modules/integrations.